### PR TITLE
Frontend CSS WEB Fonts

### LINKED ISSUE #7262: Front-end: CSS Web fonts
### Description

Documentation on CSS Web fonts

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Yashesvinee**
<hr>

## Description 📜

Added documentation of CSS web fonts
Fixes #7262 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Yashesvinee** [comment]
@2024-SANDHYA

File: Web_Development/FrontEnd/CSS_web_fonts.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# CSS WEB FONTS
+
+CSS Web Fonts allow us to use fonts that are not installed locally. After choosing the the desired font, include the font file on the web server and you are good to go.
+This font can be defined within the CSS ` @font-face ` rule.
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/CSS_web_fonts.md:4]: Not the exact correct definition. Web fonts are more than this, please include more information & also mention about importing Google Fonts into HTML & CSS file

File: Web_Development/FrontEnd/CSS_web_fonts.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	<style>
+		@font-face{
+				
+			font-family:monospace;   	/* Set font family to monospace */
+			src:url(sansation_light.woff);		}
+		
+		* {
+			font-family:monospace;
+			font-style:italic;       /** font style to italic */      }
+
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/CSS_web_fonts.md:38]: Remove so many indentations, please write in a proper manner. The final output doesn't look good.

image

File: Web_Development/FrontEnd/CSS_web_fonts.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- OTF - OpenType Fonts: A font format built on TrueType that allows a single font to support multiple writing systems within the same script. Its a commonly used format in many computer platforms today.
+
+- WOFF - Web Open Font Format - A format used in web pages whose goal is to support font distribution from a server to a client in a network with bandwidth constraints. It is a compressed OpenType/TrueType with additional metadata. WOFF has a second version (WOFF2) that differs with respect to the compression algorithm used.
+
+- EOT - Embedded OpenType format - Compact OpenType fonts that are used as embedded fonts on web pages.
+
+- SVG Fonts -  Glyph information is embedded into SVGs. The SVG 1.1 version allows creation of fonts within a SVG document.
+
+ Learn more about ` @font-face ` [here](https://www.digitalocean.com/community/tutorials/css-font-face).
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/CSS_web_fonts.md:73]: Remove this, rather make a heading "reference links" & under that mention the website.

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
Please do the needful & reopen the same PR closing this for now so you can work.

**Review by 2024-SANDHYA — APPROVED** [review]

**Pull request merged**